sources/Cartridge/Cartridge.cpp

```diff
@@ -156,10 +158,30 @@ namespace ComSquare::Cartridge
 		return bestAddress;
 	}
 
+	bool Cartridge::_isSPCFile()
+	{
+		std::string str = std::string(reinterpret_cast<char *>(this->_data), 0x21);
+
+		if (str != "SNES-SPC700 Sound File Data v0.30")
```

Comment:
Is this string a magic string to confirm the header file?
If it is, you could make that a constexpr.